bank-accounts.rb

```diff
@@ -1,18 +1,22 @@
 require 'csv'
+require 'money'
+require 'colorize'
+I18n.enforce_available_locales = false
 
 module Bank
 
   class Owner
 
-    attr_accessor :id, :last, :first, :street, :city, :state
+    attr_accessor :id, :last, :first, :street, :city, :state, :accounts
 
-    def initialize(id, last, first, street, city, state)
+    def initialize(id, last, first, street = nil, city = nil, state = nil)
       @id = id.to_i
       @first = first
       @last = last
       @street = street
       @city = city
       @state = state
+      @accounts = {}
     end
 
     def self.all
```

Comment:
Nice job using the `all` within the `find` below in order to write the `all` logic only once